packages/extension-base/src/utils/account/transform.ts

```diff
@@ -597,6 +616,10 @@ export const _combineAccounts = (accounts: AccountJson[], modifyPairs: ModifyPai
             accountActions = accountActions.filter((action) => action !== AccountActions.DERIVE);
           }
 
+          if (chainTypes.length === 1 && accountActions.includes(AccountActions.EXPORT_MNEMONIC) && account.type !== 'ton-native') {
```

Comment:
is there another way to detect solo account?

Reply:
Can also add condition to check if account is master account, `account.isMasterAccount = true`

Reply:
I found no more infomations to check

Reply:
I have updated this ✅